src/ui/screens/Parachains/ParachainDetailScreen.tsx

```diff
@@ -133,19 +68,19 @@ export function ParachainDetailScreen({route}: ScreenProps) {
                 left={() => <LeftIcon icon="sync" />}
                 right={() => (
                   <View style={styles.accessoryRight}>
-                    <Text>{parachainInfo?.lifecycle?.toString()}</Text>
+                    <Text>{parachain.lifecycle}</Text>
                   </View>
                 )}
               />
-              <Divider />
+              {parachain.homepage ? <Divider /> : null}
```

Comment:
You can wrap the Divider and the Button in a fragment or a View so you don't check this twice:
```
<>
  <Divider />
  <Button .... />
</>
```